### ISSUE TITLE
RTFD builds are broken for 4 months

### ISSUE THREAD

**Issue by hugsy** [hugsy/gef#471]
Problem in `mkdocs` ? 

[Example of failed build](https://readthedocs.org/api/v2/build/9358249.txt)

Also the RTFD label doesn't reflect the correct state of the builds.

**Comment by hugsy** [comment]
As an update, the issue does come from RTFD, not us:

After applying commit  #1f11f56 , `mkdocs` runs without a problem locally (with the same old version RTFD uses - 0.17.3)

```bash
$ python3 -m mkdocs build --verbose --clean --site-dir _build/html --config-file mkdocs.yml
DEBUG   -  Loading configuration file: <_io.BufferedReader name='mkdocs.yml'> 
DEBUG   -  Loaded theme configuration for 'readthedocs' from '/home/hugsy/.local/lib/python3.6/site-packages/mkdocs/themes/readthedocs/mkdocs_theme.yml': {'static_templates': ['404.html'], 'include_search_page': True, 'search_index_only': False} 
DEBUG   -  Config value: 'config_file_path' = 'mkdocs.yml' 
DEBUG   -  Config value: 'site_name' = 'GEF - GDB Enhanced Features documentation' 
DEBUG   -  Config value: 'pages' = [{'Home': 'index.md'}, {'Configuration': 'config.md'}, {'Frequently Asked Questions': 'faq.md'}, {'Extending GEF': 'api.md'}, {'Features': 'commands.md'}, {'Screenshots': 'screenshots.md'}, {'Command aliases': 'commands/aliases.md'}, {'Command aslr': 'commands/aslr.md'}, {'Command assemble': 'commands/assemble.md'}, {'Command canary': 'commands/canary.md'}, {'Command capstone-disassemble': 'commands/capstone-disassemble.md'}, {'Command checksec': 'commands/checksec.md'}, {'Command config': 'commands/config.md'}, {'Command context': 'commands/context.md'}, {'Command dereference': 'commands/dereference.md'}, {'Command edit-flags': 'commands/edit-flags.md'}, {'Command elf-info': 'commands/elf-info.md'}, {'Command entry-break': 'commands/entry-break.md'}, {'Command eval': 'commands/eval.md'}, {'Command format-string-helper': 'commands/format-string-helper.md'}, {'Command functions': 'commands/functions.md'}, {'Command gef-remote': 'commands/gef-remote.md'}, {'Command got': 'commands/got.md'}, {'Command heap-analysis-helper': 'commands/heap-analysis-helper.md'}, {'Command heap': 'commands/heap.md'}, {'Command help': 'commands/help.md'}, {'Command hexdump': 'commands/hexdump.md'}, {'Command highlight': 'commands/highlight.md'}, {'Command hijack-fd': 'commands/hijack-fd.md'}, {'Command ida-interact': 'commands/ida-interact.md'}, {'Command is-syscall': 'commands/is-syscall.md'}, {'Command ksymaddr': 'commands/ksymaddr.md'}, {'Command memory': 'commands/memory.md'}, {'Command name-break': 'commands/name-break.md'}, {'Command nop': 'commands/nop.md'}, {'Command patch': 'commands/patch.md'}, {'Command pattern': 'commands/pattern.md'}, {'Command pcustom': 'commands/pcustom.md'}, {'Command pie': 'commands/pie.md'}, {'Command print-format': 'commands/print-format.md'}, {'Command process-search': 'commands/process-search.md'}, {'Command process-status': 'commands/process-status.md'}, {'Command registers': 'commands/registers.md'}, {'Command reset-cache': 'commands/reset-cache.md'}, {'Command ropper': 'commands/ropper.md'}, {'Command scan': 'commands/scan.md'}, {'Command search-pattern': 'commands/search-pattern.md'}, {'Command set-permission': 'commands/set-permission.md'}, {'Command shellcode': 'commands/shellcode.md'}, {'Command stub': 'commands/stub.md'}, {'Command syscall-args': 'commands/syscall-args.md'}, {'Command theme': 'commands/theme.md'}, {'Command tmux-setup': 'commands/tmux-setup.md'}, {'Command trace-run': 'commands/trace-run.md'}, {'Command unicorn-emulate': 'commands/unicorn-emulate.md'}, {'Command vmmap': 'commands/vmmap.md'}, {'Command xfiles': 'commands/xfiles.md'}, {'Command xinfo': 'commands/xinfo.md'}, {'Command xor-memory': 'commands/xor-memory.md'}] 
DEBUG   -  Config value: 'site_url' = '' 
DEBUG   -  Config value: 'site_description' = None 
DEBUG   -  Config value: 'site_author' = None 
DEBUG   -  Config value: 'theme' = Theme(name='readthedocs', dirs=['/home/hugsy/.local/lib/python3.6/site-packages/mkdocs/themes/readthedocs', '/home/hugsy/.local/lib/python3.6/site-packages/mkdocs/templates'], static_templates=['sitemap.xml', '404.html'], include_search_page=True, search_index_only=False) 
DEBUG   -  Config value: 'docs_dir' = '/home/hugsy/code/gef/docs' 
DEBUG   -  Config value: 'site_dir' = '/home/hugsy/code/gef/_build/html' 
DEBUG   -  Config value: 'theme_dir' = None 
DEBUG   -  Config value: 'copyright' = None 
DEBUG   -  Config value: 'google_analytics' = None 
DEBUG   -  Config value: 'dev_addr' = Address(host='127.0.0.1', port=8000) 
DEBUG   -  Config value: 'use_directory_urls' = True 
DEBUG   -  Config value: 'repo_url' = '' 
DEBUG   -  Config value: 'repo_name' = '' 
DEBUG   -  Config value: 'edit_uri' = '' 
DEBUG   -  Config value: 'extra_css' = [] 
DEBUG   -  Config value: 'extra_javascript' = [] 
DEBUG   -  Config value: 'extra_templates' = [] 
DEBUG   -  Config value: 'markdown_extensions' = ['toc', 'tables', 'fenced_code'] 
DEBUG   -  Config value: 'mdx_configs' = {} 
DEBUG   -  Config value: 'strict' = False 
DEBUG   -  Config value: 'remote_branch' = 'gh-pages' 
DEBUG   -  Config value: 'remote_name' = 'origin' 
DEBUG   -  Config value: 'extra' = {} 
DEBUG   -  Config value: 'plugins' = PluginCollection([('search', <mkdocs.contrib.legacy_search.SearchPlugin object at 0x7f5015a5c6d8>)]) 
INFO    -  Cleaning site directory 
INFO    -  Building documentation to directory: /home/hugsy/code/gef/_build/html 
DEBUG   -  Copying static assets from /home/hugsy/.local/lib/python3.6/site-packages/mkdocs/contrib/legacy_search/templates 
DEBUG   -  Copying static assets from the docs dir. 
DEBUG   -  Building template: sitemap.xml 
DEBUG   -  Building extra_templates pages 
DEBUG   -  Building markdown pages. 
DEBUG   -  Building page index.md 
```

Looking into https://github.com/readthedocs/readthedocs.org/issues/5474 for workarounds..

**Comment by hugsy** [comment]
Fixed by  #fdfb509
